Lib/site.py

```diff
@@ -390,14 +394,18 @@ def getsitepackages(prefixes=None):
 
         implementation = _get_implementation().lower()
         ver = sys.version_info
+        if hasattr(sys, 'abiflags') and 't' in sys.abiflags:
+            abi_thread = 't'
+        else:
+            abi_thread = ''
         if os.sep == '/':
             libdirs = [sys.platlibdir]
             if sys.platlibdir != "lib":
                 libdirs.append("lib")
 
             for libdir in libdirs:
                 path = os.path.join(prefix, libdir,
-                                    f"{implementation}{ver[0]}.{ver[1]}",
+                                    f"{implementation}{ver[0]}.{ver[1]}{abi_thread}",
```

Comment:
@colesbury -- would you be able to update the [docs for the `site` module](https://docs.python.org/3.13/library/site.html) to reflect these changes to how Python finds the `site-packages` directory?

(The context is I'm currently working on reimplementing Python's logic here for Astral's work-in-progress type checker. I was reading through the `site.py` source code, and it surprised me to find logic here that wasn't documented!)

Reply:
Thanks for the pointer. I put up a PR to update the docs: https://github.com/python/cpython/pull/122737